### PR TITLE
Feat/more css

### PR DESCRIPTION
**Pull request by mattsaqueli**
Added FitLit logo and adjusted widget CSS.

### CODE REVIEW

File: dist/assets/fitlit-logo.png
Comment (sarahcatherine311) [inline dist/assets/fitlit-logo.png:1]: Love the Logo!!

File: src/scripts.js [hunk 1]
```diff
@@ -198,13 +205,15 @@ userNoteBtn.addEventListener('submit', (event) => {
...
+    const userEntries = activityNotes.filter(entry => entry.userID === newUser.id)
+    userEntries.forEach(entry => {
+      activityNote.innerText += ` ${entry.date}: ${entry.activityInput}
+  
+      `
+    })
+  }
 };
 
 //On userNoteBtn add an event listener that incokes a function addUserNote(newUser.id)
```
Comment (sarahcatherine311) [inline src/scripts.js:219]: We can go back in and remove commented out code and clean up syntax next

File: src/scripts.js [hunk 2]
```diff
@@ -198,13 +205,15 @@ userNoteBtn.addEventListener('submit', (event) => {
...
 
 function displayActivityNote() {
-  activityNotes = JSON.parse(localStorage.getItem("activityNotes")) 
-  const userEntries = activityNotes.filter(entry => entry.userID === newUser.id)
-  userEntries.forEach(entry => {
-    activityNote.innerText += ` ${entry.date}: ${entry.activityInput}
-
-    `
-  })
+  if (localStorage.getItem("activityNotes")) {
```
Comment (Devynnem) [inline src/scripts.js:208]: If statement for the local storage win!

**Review by sarahcatherine311 — APPROVED** [review]
Nice work team! Looks great.

**Review by tdmburr — COMMENTED** [review]
Looks great. Great work on logo additions, local storage, etc.

**Pull request merged**